reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -411,13 +411,15 @@ def initialize_traefik(self):
         Traefik dashboard service is not accessible by default, to make it
         accessible inside Minikube service type is changed to NodePort.
         """
-        from reana_cluster.config import traefik_configuration_file_path
+        from reana_cluster.config import (traefik_configuration_file_path,
```

Comment:
Imports are done here because of circular dependencies (config.py imports KubernetesBackend class). I have no idea how to solve it in a better way. Suggestions are welcome :)

Reply:
Something we could do (only if this becomes worse) is how we do in Invenio, loading either and object or importing using the string value, see [obj_or_import_string](https://github.com/inveniosoftware/invenio-accounts-rest/blob/a40a602dc4239302bc5ba4a95ff9d6d85f100aa5/invenio_accounts_rest/utils.py#L32), how we set in it [config](https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/config.py#L35) and how we [load it](https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/ext.py#L31). Though, I think for now we can go like this for now.